com.microsoft.mrtk.core/Utilities/XRSubsystemHelpers.cs

```diff
@@ -151,6 +154,23 @@ public static XRDisplaySubsystem DisplaySubsystem
             }
         }
 
+        private static HandsAggregatorSubsystem handsAggregator = null;
+
+        /// <summary>
+        /// The currently loaded and running hands aggregator, if any.
+        /// </summary>
+        public static HandsAggregatorSubsystem HandsAggregator
```

Comment:
Ah, I have basically this exact change in a local branch too! I put it in `HandsUtils` though, potentially as a replacement for 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/1405fd95fdf06652207c7c50e2e6e0c4586f5309/com.microsoft.mrtk.core/Utilities/HandsUtils.cs#L16-L25

Do you think it makes more sense here (and either way, we should probably deprecate `HandsUtils.GetSubsystem()`) and we might expect to fill this class out with _all_ of our subsystem types?

Reply:
I actually forgot about the existence of this utils class....

I'm ambivalent. Given that these are basically just shorthand aliases for the equivalent `GetFirstRunningSubsystem<T>`, it doesn't really matter much where it is.

Reply:
I can just stick this in HandsUtils I suppose.

Reply:
But the HandsUtils one already uses the method that is now memoized/optimized here, so... no huge difference

Reply:
I'll deprecate the HandsUtils one.

Reply:
Wew lad that had a lot of knock-on effects. Check commit